src/main/java/gov/nasa/cumulus/gov/nasa/cumulus/bo/ExtraFileFileds.java

```diff
@@ -0,0 +1,33 @@
+package gov.nasa.cumulus.gov.nasa.cumulus.bo;
```

Comment:
Can you help me understand the package namespace of gov.nasa.cumulus.gov.nasa.cumulus.bo? Are we copying over from cumulus package ExtraFileFileds class or something? If so, can we import the cumulus package directly?

Reply:
bo means business logic and it is just a POJO being used to "carry data from the decoder to the main lambda function".
The decoder is for code isolation so we have logic of decoding uri to filename, bucket on the decoder and the BO is for a way of encapsulating "data carriage" by returning 2 things (filename, bucket)